### ISSUE TITLE
Size of windows preview when alt-tab

### ISSUE THREAD

**Issue by MattOrmianek** [lwouis/alt-tab-macos#3461]
Problem with weird sizes of windows when preview is on when alt-tab. Showed on screens. When I restart AltTab sizes are recalculated (I guess) and everything works, but when I change app on what screen is or just unplug external monitor everything bugs. 
On macbook everything works properly (only sometimes, also bugs sometimes there)
On other screens with bug.
When a bug occurs on one screen, it also appears on the others.
![IMG_7759](https://github.com/lwouis/alt-tab-macos/assets/16574489/ed0f10f6-addc-4ab4-a724-86a5ecca9269)
![IMG_7758](https://github.com/lwouis/alt-tab-macos/assets/16574489/9c8cfcd8-bd14-4158-a779-cf5c8fd71270)
![arc_error](https://github.com/lwouis/alt-tab-macos/assets/16574489/2cc23afd-3f82-4b12-958f-3d93a27d0322)

1. Turn on 'Preview selected window' option in AltTab
2. Faster way: unplug external monitor and press AltTab shortcut
3. Slower way: try moving widows on other spaces on mac and pressing AltTab shortcut

* AltTab version: 6.69.0
* macOS version: 14.5 (23F79)
* Other relevant info: external monitor (but not necessary)

**Comment by lwouis** [comment]
Hi,

I think this bug may have been fixed in v7.2.0.

Could you please test it and let us know if it's fixed?

Thank you 🙇

**Comment by MateuszOrmianek** [comment]
Well, after today update app crashed once and I got one bugged screen. It was different this time, I mean problem was same but it was looking different, not like anything before.

**Comment by MattOrmianek** [comment]
Sorry for responding from another account, if you need help with that how to remake this problem I can help with that.

**Comment by lwouis** [comment]
Yes, please. Could you please share step-by-step how you reproduce the issue? A video would help too 🙇

**Comment by MattOrmianek** [comment]
Since last update I got 2 types of problems. Today I wanted to record video and show you how to recreate this bug. But something strange is happening. I am not sure in 100% if problem is with AltTab, but tbh this is only software I updated recently. When I wanted to recreate bug, I noticed strange behaviour, I saw bugged screen size, but in like 2-3 seconds it fixed.

But there is another bug, when I unplug monitor sometimes something really strange is going on with macbook. I can't see mission control, can't see my dock, can't change virtual desktops (only via custom keyboard shortcut, not working via touchpad),  just alttab is working and I can change apps via alttab. When I plug my monitor back everything is working fine. I don't know if my macbook is cursed or sth. 

I am trying to find a way for recreating bug with windows size but I have a lot of things to do. When I get how to recreate it, I will record a video. 

Can you help me if AltTab is doing weird things with my macbook, with dock, mission control etc, did you get any tickets similar to this problems?

**Comment by lwouis** [comment]
> Can you help me if AltTab is doing weird things with my macbook, with dock, mission control etc, did you get any tickets similar to this problems?

You're the first one to mention such issue. It may be AltTab, but it could also be some issue with macOS and your particular external monitor. I'm not sure. I need to be able to reproduce the issue, otherwise debugging is quite impossible 🙇

**Comment by MattOrmianek** [comment]
Sure, in that case I will try to record what's wrong and how to recreate it. Maybe I will use some keyboard input monitoring method to do that because it is random as hell. Thanks for response.

**Comment by lwouis** [comment]
I think this issue will be fixed in the next release

**Comment by MattOrmianek** [comment]
Nice, thanks. Sorry for not responding, a lot of things to do.
